source/jormungandr/jormungandr/street_network/tests/geovelo_test.py

```diff
@@ -434,3 +434,24 @@ def isochrone_geovelo_test():
         assert geovelo_response.rows[0].routing_response[0].routing_status == response_pb2.reached
         assert geovelo_response.rows[0].routing_response[1].duration == 1656
         assert geovelo_response.rows[0].routing_response[1].routing_status == response_pb2.reached
+
+def distances_durations_test():
+    """
+    Check that the response from geovelo is correctly formatted with 'distances' and 'durations' sections
+    """
+    instance = MagicMock()
+    geovelo = Geovelo(instance=instance,
+                      service_url='http://bob.com')
+    resp_json = direct_path_response_valid()
+
+    origin = make_pt_object(type_pb2.ADDRESS, lon=2, lat=48.2, uri='refStart1')
+    destination = make_pt_object(type_pb2.ADDRESS, lon=3, lat=48.3, uri='refEnd1')
+    fallback_extremity = PeriodExtremity(str_to_time_stamp('20161010T152000'), True)
+
+    proto_resp = geovelo._get_response(resp_json, origin, destination, fallback_extremity)
+    assert proto_resp.journeys[0].durations.total
```

Comment:
I don't understand theses lines

Reply:
Yep, I think the next line is enough in python.
You can probably just add 3 lines in `direct_path_geovelo_test` and `direct_path_geovelo_zero_test` to test that, and put real values (not only `!=0`) so that the response consistency is checked (length of section vs. journey, etc.)

Reply:
These 2 lines are useful IMO. In the ProtoResponse, a field is not present at all if it isn't set. So the 2 lines are there to highlight 2 different problems : is the field set and does it have a value? 
Second point, I agree that to put a value will be better than the current assert but the real duration and distance needs to be calculated, I won't just put the value returned and believe it true. Also, a small change in the data (for future tests for ex.) will force us to calculate it again. Not worth the effort!